src/main/pages/features/Features.js

```diff
@@ -0,0 +1,117 @@
+    playCircle: PlayCircleIcon
+  };
+
+  const BannerBox = ({title, icon, paragraph, link}) => {
+    return (
+      <div className="features-banner__list__item-details">
+        <ImageIcon
+          className="features-banner__list__item-details__main-icon"
+          icon={icons[icon]}
+          label={title}
+          title={title}
+        />
+        <h5 className="features-banner__list__item-details__title">{ title }</h5>
+        <p className="features-banner__list__item-details__paragraph">{ paragraph }</p>
+        <Action to={link} includeArrow={true} className="features-banner__list__item-details__learn-button">
+          <div className="features-banner__list__item-details__learn-button__text">Learn More</div>
+        </Action>
+      </div>
+    );
+  };
+
+  const TenanciesList = ({monthly=false}) => {
+    return (
+      <div className="tenancies-list">
+        <TenancyInfo
+          tenancy="PAY_AS_YOU_GO"
+          level="Level 1"
+          service="Self Service"
+          features={["ALL_MEDIA_PLATFORM", "ALL_CONTENT_DISTRIBUTION", "ALL_WEB3"]}
+          monthlyPrice="0"
+          additionalCostText="+ utility fees"
+        />
+        <TenancyInfo
+          tenancy="ADVANCED"
+          level="Level 2"
+          service="Tier 1 Support"
+          features={["ALL_MEDIA_PLATFORM", "ALL_CONTENT_DISTRIBUTION", "ALL_WEB3"]}
+          monthlyPrice={monthly ? "995" : "833"}
+          additionalCostText={monthly ? "+ utility fees" : "+ utility fees in annual commit"}
+          addedBenefitPercentage="50+%"
+          addedBenefitText="On selected Content Fabric utility rates"
+        />
+        <TenancyInfo
+          tenancy="ENTERPRISE"
+          level="Level 3"
+          service=" Full Valet Service"
+          features={["ALL_MEDIA_PLATFORM", "ALL_CONTENT_DISTRIBUTION", "ALL_WEB3"]}
+          monthlyPrice={monthly ? "9,995" : "8,333"}
+          additionalCostText={monthly ? "+ utility fees" : "+ utility fees in annual commit"}
+          addedBenefitPercentage="90+%"
+          addedBenefitText="On selected Content Fabric utility rates"
+        />
+      </div>
+    );
+  };
+
+  return (
+    <div className="page">
+      <div className="page__header-container">
+        <h1>Eluvio Tenancies: Your Secure, Private Web3 Space</h1>
+        <h3>What works best for your brand?</h3>
+      </div>
+      <div className="page__content-block">
+        <TabbedInfoBox
+          noBackgroundStyling={true}
+          tabs={[
+            {
+              title: "Monthly",
+              content: TenanciesList({monthly: true})
+            },
+            {
+              title: "Annually",
+              content: TenanciesList({monthly: false})
+            }
+          ]}
+        />
+
+        <div className="features-connect">
+          <Button className="light primary features-connect__button">Connect with us</Button>
+        </div>
+
+        <div className="features-banner">
```

Comment:
The banner and the faq are supposed to be full-bleed according to the design. I alluded to this when we talked, but I think what we can do is in MainApp.js -> PageContainer, we can add parameters for before/after content that is not within the padded block.

```
const PageContainer = observer(({children, before, after, padded=false, dark=false, unbound=false}) => {
 ....

  return (
    <>
      <Header />
      { before }
      <div
        key={`page-container-${location.pathname}`}
        className={`page-container fade-in ${padded ? "padded-block" : ""} ${dark ? "dark" : "light"} ${unbound ? "unbound" : ""}`}
      >
        {children}
      </div>
      { after }
      <ContactForm dark={dark} />
      <Footer dark={dark} />
    </>
  );
});
```

alternatively, and actually I think this is a better idea, specify this page as unpadded, then add 'padded-block' div around the stuff on the page that's supposed to be padded.

Reply:
Haha I actually was only adding the padding in a media query for tablet breakpoint but I'll add `padded-block` to the html for those elements

Reply:
Oh I didn't notice the `max-width: 1600px` on the `page-container`. I'll just add the full width components to `after`